app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -74,9 +75,16 @@ class SettingsViewModel @Inject constructor(
         command.value = Command.LaunchFeedback
     }
 
-    fun userRequestedToChangeAutocompleteSetting(checked: Boolean) {
-        Timber.i("User changed autocomplete setting, is now enabled: $checked")
-        settingsDataStore.autoCompleteSuggestionsEnabled = checked
+    fun onLightThemeToggled(enabled: Boolean) {
+        Timber.i("User toggled light theme, is now enabled: $enabled")
+        settingsDataStore.lightThemeEnabled = enabled
+        command.value = Command.UpdateTheme
+        command.value = null
```

Comment:
This seems a little odd... why is this needed?

Reply:
When recreate is called and the last command fires again

Reply:
Commented elsewhere, but it's probably hard to find in this UI....

Maybe instead of `command.value = null` we should make use of `SingleLiveEvent` for the `Commands`. We do this in other activities.

    val command: MutableLiveData<Command> = MutableLiveData()

becomes

    val command: SingleLiveEvent<Command> = SingleLiveEvent()